### ISSUE TITLE
Missing documentation for test-campaign

### ISSUE THREAD

**Issue by mfaure** [Asqatasun/Asqatasun#147]
## Observed behaviour

Documentation for running a test campaign is not written

## Expected behaviour

Have a documentation for how to run a test-campaign. Having it clear and eaisly understandable  would be great :)

**Comment by dzc34** [comment]
@mfaure, can you explain "test campaign"?

**Comment by mfaure** [comment]
I meant krashtest campaign or smoke test campaign: run hundreds of audit on an Asqatasun instance to make sure everything is fine, even after lots of user actions.

See work in progress https://github.com/Asqatasun/Asqatasun/commits/147-doc-test-campaign

**Comment by dzc34** [comment]
OK

for `nonreg.asqatasun.org `
don't forget to adding a `robots.txt` file 
```
    User-agent: *
    Disallow: /

    User-Agent: asqatasun
    Allow: /
```

**Comment by mfaure** [comment]
done

**Comment by dzc34** [comment]
not done yet!
I revert your commit because it break the build.
But I missing to adding a comment here.

**Comment by mfaure** [comment]
@dzc34 as far as I see, the branch is actually merged in develop and Travis is OK, isn't it ?

**Comment by dzc34** [comment]
no, but...
![selection_001](https://user-images.githubusercontent.com/6709977/30762464-6b8dbfc2-9fe2-11e7-823c-fd52cba00dfb.png)

**Comment by dzc34** [comment]
2 solutions (tested) : 

- revert the revert commit c03c9b62e86cf6223f2838b072c880c380f04209 !!
- chery Pick each commits

**Comment by mfaure** [comment]
oups sorry, OK

**Comment by dzc34** [comment]
use the new branch **147b-doc-krashtest-campaign_V2**

**Comment by mfaure** [comment]
closed by merging #177
